server/fleet/hosts.go

```diff
@@ -201,7 +201,7 @@ type HostListOptions struct {
 // TODO(Sarah): Are we missing any filters here? Should all MDM filters be included?
 func (h HostListOptions) Empty() bool {
 	return h.ListOptions.Empty() &&
-		h.DeviceMapping == false &&
+		h.DeviceMapping &&
```

Comment:
Are you sure that's correct? I think this `Empty` method is basically "check if h is the zero value of `HostListOptions`"? (which I never understood why we didn't just do the check against its zero value). This (and `DisableFailingPolicies` below) changes the meaning of `Empty`.

Reply:
@mna great catch, thanks! this is just me being dumb 🤦 , I wanted to do this:

```diff
- h.DeviceMapping == false 
+ !h.DeviceMapping 
```

I will just revert it back for now